les/vflux/client/serverpool.go

```diff
@@ -307,6 +338,9 @@ func (s *ServerPool) RegisterNode(node *enode.Node) (*NodeValueTracker, error) {
 	s.ns.SetState(node, sfConnected, sfDialing.Or(sfWaitDialTimeout), 0)
 	nvt := s.vt.Register(node.ID())
 	s.ns.SetField(node, sfiConnectedStats, nvt.RtStats())
+	if node.IP().IsLoopback() {
+		s.ns.SetField(node, sfiLocalAddress, node.Record())
+	}
```

Comment:
It does not make any difference here (these fields don't have an immediate effect elsewhere, no one will read them before the register function returns) but I changed it so that it looks more consistent now :)